frontend/src/App.tsx

```diff
@@ -14,6 +19,8 @@ const App = (props: RouteProps) => {
         <Route path='/Login' component={Login} />
         <Route exact path='/postreg' component={PostRegister} />
         <Route exact path='/dashboard' component={Houses} />
+        <Route exact path='/settings' component={Settings} />
+        <Route exact path='/updateinfo' component={PostRegister} />
```

Comment:
Login.tsx still pushes to /postreg, that should be changed.

Reply:
The `PostRegister` component does different behavior based on the value of `props.location.pathname`. If it's rendered through `/postreg`, it works as it has before. If it's rendered through `/updateinfo`, current user data will be populated into the fields.

I should also condition the redirect based on the location pathname as well. I can do an additional commit for that.

Reply:
Ahhhh, sorry missed that!

Reply:
The more I think about it though, having two separate paths might not be necessary. Should I do a quick change? Maybe I can save it when I refactor the PostRegister component in the weekend/early next week (because it is currently a mountain of crazy code).

Reply:
Think we should kick that can down the road and go with the functionality for now!

Reply:
If we're planning on refactoring this soon, let's not worry about changing it right now 😁